### PR TITLE
Add integration test for handling missing project metadata

### PR DESCRIPTION
**Pull request by slimreaper35**
Test case if there no setup files in the repository or cachi2 fails to resolve the name.

JIRA: https://issues.redhat.com/browse/STONEBLD-1181
REPO: https://github.com/cachito-testing/cachito-pip-without-metadata

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- n/a Code coverage from testing does not decrease and new code is covered
- n/a Docs updated (if applicable)
- n/a Docs links in the code are still valid (if docs were updated)

### CODE REVIEW

**Comment by chmeliik** [comment]
The commit title seems too generic, the commit adds one specific test case

Other than that, LGTM :+1:

**Review by taylormadore — APPROVED** [review]
LGTM :+1:

File: tests/integration/test_pip.py [hunk 1]
```diff
@@ -66,6 +66,20 @@
             ),
             id="pip_local_path",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachito-pip-without-metadata.git",
```
Comment (lkolacek) [inline tests/integration/test_pip.py:71]: Should we add a simple Readme to source repository and explain there what's missing and how Cachi2 should react?

Comment (slimreaper35) [reply tests/integration/test_pip.py:71]: Okay, why not

**Review by chmeliik — APPROVED** [review]

File: tests/integration/test_pip.py (hunk 1 above)
Comment (lkolacek) [reply tests/integration/test_pip.py:71]: thx looks great

**Review by lkolacek — APPROVED** [review]
LGTM :+1:

**Pull request merged**